src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -17,7 +16,7 @@ class FeedbackDao(private val sql: DSLContext) {
             record[feedbackTable.employeeId],
             record[feedbackTable.companyId],
             record[feedbackTable.dateOfFeedback],
-            AnonymityType.convertStringToAnonymityType(record[feedbackTable.anonymity]),
+            AnonymityType.convertToAnonymityType(record[feedbackTable.anonymity]),
```

Comment:
so because this is the enum, unless you specified something else, when you say 'convertTo' it is clear that you are converting to anonymityType. what is not clear, is what you are converting from.
So it's better to rename it to "fromString", "convertFromString", something like that